arctic/chunkstore/chunkstore.py

```diff
@@ -110,6 +123,9 @@ def read(self, symbol, chunk_range=None):
         chunk_range: object
             corresponding range object for the specified chunker (for
             DateChunker it is a DateRange object)
+        no_filter: boolean
+            perform chunk level filtering on the data (see filter() in _chunker)
```

Comment:
1. testing
2. its also used when deleting a range. If you filter out the data during a read, you can't preserve parts of the chunk that you might want to save. (ex: if the delete range is March 15 - April 15 and your chunking strat is monthly. You need to not filter the chunks when reading so you can preserve the March 1-14 data and the April 16-30 data).